specs/src/specs/params.md

```diff
@@ -0,0 +1,50 @@
+# Celestia Governance Params
+
+These are the parameters for mainnet. Note that not all of these parameters are
+changable via governance. This list also includes parameter that require a
+hardfork to change due to being manually hardcoded in the application or they
+are blocked by the `x/paramfilter` module.
+
+## Parameters
+
+| Module.Parameter | Default | Summary | Changeable via Governance |
+| --- | --- | --- | --- |
+| auth.MaxMemoCharacters | 256 | Largest allowed size for a memo in bytes. | True |
+| auth.TxSigLimit | 7 | Max number of signatures allowed in a multisig transaction. | True |
+| auth.TxSizeCostPerByte | 10 | Gas used per transaction byte. | True |
+| auth.SigVerifyCostED25519 | 590 | Gas used to verify Ed25519 signature. | True |
+| auth.SigVerifyCostSecp256k1 | 1000 | Gas used to verify secp256k1 signature. | True |
+| bank.SendEnabled | true | Allow transfers. | False |
+| blob.GasPerBlobByte | 8 | Gas used per blob byte. | True |
+| MaxBlockBytes | 100MiB | Hardcoded value in CometBFT for the protobuf encoded block. | False |
+| MaxSquareSize | 128 | Hardcoded value in the applications that requires a hardfork to change. | False |
+| blob.GovMaxSquareSize | 64 | Governance parameter for the  square size. If larger than MaxSquareSize, MaxSquareSize is used. | True |
+| consensus.block.MaxBytes | 1.88MiB | Governance parameter for the maximum size of the block. | True |
```

Comment:
Can you add a detail of what "size of the block" refers to here? could mean EDS/ODS/raw block data in bytes...

Reply:
:+1: 992fb4e130692c908625940642aba7d63f95d2f4